composer.json

```diff
@@ -7,8 +7,18 @@
         "php": ">=5.4.0",
         "react/promise": "~2.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.5"
```

Comment:
Not sure if we ever reached a consensus on this..
*Personally*, I think it makes sense to explicitly define *all* dependencies - however this is something we've discussed in the past and have chosen to *not* do this for the other components.
As such, I'd recommend to keep this for a separate PR/discussion so that we can get this PR in earlier and am :-1: (on only this line) for now.